frontend/packages/dev-console/src/components/NamespacedPage.tsx

```diff
@@ -23,10 +24,14 @@ const NamespacedPage: React.FC<NamespacedPageProps> = ({
   onNamespaceChange,
   hideApplications = false,
   variant = NamespacedPageVariants.default,
+  toolbarOptions = undefined,
```

Comment:
`undefined` is never required as the RH of an equal. If it's not there, it is `undefined` and if it is there (like `null`) it won't default to `undefined`.